common/views/components/Header/Header.js

```diff
@@ -34,88 +34,81 @@ const links = [
 type Props = {|
   siteSection: string,
   isActive: boolean,
-  isFixed: boolean,
   toggle: () => void,
 |};
 
-const Header = withToggler(
-  ({ siteSection, toggle, isActive, isFixed }: Props) => (
-    <div
-      className={classNames({
-        'header grid bg-white border-color-pumice border-bottom-width-1': true,
-        'header--is-burger-open': isActive,
-      })}
-      style={{
-        height: '85px',
-        position: isFixed ? 'fixed' : 'static',
-        top: 0,
-        left: 0,
-        right: 0,
-      }}
-    >
-      <span className="visually-hidden">reset focus</span>
-      <div className="header__upper grid__cell">
-        <div className="header__inner container">
-          <div className="header__burger">
-            <a
-              href="#footer-nav-1"
-              id="header-burger-trigger"
-              className="header__burger-trigger"
-              aria-label="menu"
-              onClick={event => {
-                event.preventDefault();
-                toggle();
-              }}
-            >
-              <span />
-              <span />
-              <span />
-            </a>
-          </div>
-          <div className="header__brand">
-            <a href="/" className="header__brand-link">
-              <WellcomeCollectionBlack />
-            </a>
-          </div>
-          <nav
-            id="header-nav"
-            className="header__nav"
-            aria-labelledby="header-burger-trigger"
+const Header = withToggler(({ siteSection, toggle, isActive }: Props) => (
```

Comment:
We've removed withToggler elsewhere if we'd like to remove it here.
Bikeshedding though.